### ISSUE TITLE
Not sure what i am doing wrong.

### ISSUE THREAD

**Issue by LORDHEDON** [Kaltharos/RPGMods#38]
So i used Thunderstore Mod Manager to download RPGMods and I have followed the instructions but i can't seem to get any of the console commands to work. Any ideas on why that could be?

**Comment by Thazzeus** [comment]
What console commands are you trying to do for example?

**Comment by UberKiller** [comment]
yeah same, neither the waypoints nor the experience commands work,
